docs/content/_docs/query_language.html

```diff
@@ -244,52 +202,175 @@ <h4>Operators</h4>
   </tr>
 
   <tr>
-    <th scope="row">Tag Starts With</th>
-    <td><pre><code class="language-hql">&lt;a&gt; ^ &lt;b&gt;</code></pre></td>
-    <td><pre><code class="language-json">["^", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
+    <th scope="row">Key Matches</th>
+    <td><pre><code class="language-hql">$key = &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["key", &lt;a&gt;]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Key Does Not Match</th>
+    <td><pre><code class="language-hql">$key != &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["not", ["key", &lt;a&gt;]]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Tag True</th>
+    <td><pre><code class="language-hql">[true]</code></pre></td>
+    <td><pre><code class="language-json">[true]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Tag False</th>
+    <td><pre><code class="language-hql">[false]</code></pre></td>
+    <td><pre><code class="language-json">[false]</code></pre></td>
+  </tr>
+  
+  <tr>
+    <th scope="row">RegEx On Tag</th>
+    <td><pre><code class="language-hql">&lt;a&gt; ~ &lt;b&gt;</code></pre></td>
+    <td><pre><code class="language-json">["~", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
+  </tr>
+  
+  <tr>
+    <th scope="row">Raw</th>
+    <td>Custom filters</td>
```

Comment:
the `custom filters` cell messes up the table formatting

![2019-10-03_092918_782935915](https://user-images.githubusercontent.com/3689782/66130872-74594600-e5c0-11e9-8a59-0dbeb9030501.png)